packages/mgt-components/src/components/mgt-file-list/mgt-file-list.ts

```diff
@@ -58,41 +58,26 @@ registerFluentComponents(fluentProgressRing, fluentDesignSystemProvider);
- * @cssprop --file-upload-border- {String} File upload border top style
- * @cssprop --file-upload-background-color - {Color} File upload background color with opacity style
- * @cssprop --file-upload-button-float - {string} Upload button float position
- * @cssprop --file-upload-button-background-color - {Color} Background color of upload button
- * @cssprop --file-upload-dialog-background-color - {Color} Background color of upload dialog
- * @cssprop --file-upload-dialog-content-background-color - {Color} Background color of dialog content
- * @cssprop --file-upload-dialog-content-color - {Color} Color of dialog content
- * @cssprop --file-upload-dialog-primarybutton-background-color - {Color} Background color of primary button
- * @cssprop --file-upload-dialog-primarybutton-color - {Color} Color text of primary button
- * @cssprop --file-upload-button-color - {Color} Text color of upload button
- * @cssprop --file-list-background-color - {Color} File list background color
- * @cssprop --file-list-box-shadow - {String} File list box shadow style
- * @cssprop --file-list-border - {String} File list border styles
- * @cssprop --file-list-padding -{String} File list padding
- * @cssprop --file-list-margin -{String} File list margin
- * @cssprop --file-item-background-color--hover - {Color} File item background hover color
- * @cssprop --file-item-border-top - {String} File item border top style
- * @cssprop --file-item-border-left - {String} File item border left style
- * @cssprop --file-item-border-right - {String} File item border right style
- * @cssprop --file-item-border-bottom - {String} File item border bottom style
- * @cssprop --file-item-background-color--active - {Color} File item background active color
- * @cssprop --file-item-border-radius - {String} File item border radius
- * @cssprop --file-item-margin - {String} File item margin
- * @cssprop --show-more-button-background-color - {Color} Show more button background color
- * @cssprop --show-more-button-background-color--hover - {Color} Show more button background hover color
- * @cssprop --show-more-button-font-size - {String} Show more button font size
- * @cssprop --show-more-button-padding - {String} Show more button padding
- * @cssprop --show-more-button-border-bottom-right-radius - {String} Show more button bottom right radius
- * @cssprop --show-more-button-border-bottom-left-radius - {String} Show more button bottom left radius
+ * @fires {CustomEvent<MicrosoftGraph.DriveItem>} itemClick - Fired when a user clicks on a file.
+ * it returns the file (DriveItem) details.
+ *
+ * NOTE: This component also allows customizing the tokens from mgt-file and mgt-file-upload components.
+ * @cssprop --file-list-background-color - {Color} the background color of the component.
+ * @cssprop --file-list-box-shadow - {String} the box-shadow syle of the component.
```

Comment:
Perhaps we should have the defaults in these comments for lengths non-color items?

Reply:
You mean like `--file-margin` and `--file-padding` values?

Reply:
yes.